lib/libquads.py

```diff
@@ -82,7 +84,8 @@ def __init__(self, config, statedir, movecommand, datearg, syncstate, initialize
         self.datearg = datearg
         self.logger = logging.getLogger("quads.Quads")
         self.logger.setLevel(logging.DEBUG)
-        set_hardware_service(JuniperDriver())
+        #set_hardware_service(JuniperDriver())  Taking away the hardcoding for Juniper
+        set_hardware_service(HilDriver()) #temporary hardcode to HIL
```

Comment:
Can we remove this replaced-hardcoded driver and do this from the configuration file?